<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addKeywordBtn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="keywordInput">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Add Keyword:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>520</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>논문 크롤링 Bot</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="radioGroup">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Option</string>
    </property>
    <widget class="QRadioButton" name="radioBtnAnd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>AND</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioBtnOr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>OR</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Query box</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="keywordRange">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string extracomment="검색 범위">Title</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Title &amp; Abstract</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Authors</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="queryBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <zorder>radioGroup</zorder>
   <zorder>addKeywordBtn</zorder>
   <zorder>keywordInput</zorder>
   <zorder>label</zorder>
   <zorder>progressBar</zorder>
   <zorder>label_2</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>label_3</zorder>
   <zorder>keywordRange</zorder>
   <zorder>search</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>queryBox</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
